src/docs/environment/index.mdx

```diff
@@ -112,7 +112,9 @@ Now, if you try and run `volta`, you should see some help text, meaning volta is
 node -v
 ```
 
-Volta intercepts this and automatically downloads and installs the node version in sentry's `package.json`.
+Volta intercepts this and automatically downloads and installs the node and yarn versions in sentry's `package.json`.
+
+Note that if you didn't already have node and yarn installed on your computer prior to this step, you'll need to run `volta install node@x.x.x` and `volta install yarn@x.x.x` when outside the `sentry` directory in order to use `node` and `yarn` elsewhere.
```

Comment:
Why is this necessary? Ideally I'd like to only provide documentation relevant to the dev environment. This is just how to use volta which anyone can read about at their leisure.

If there's other sentry-related repositories you needed non-volta node/yarn on, I can take a look.

Reply:
Yeah, fair, I guess. Was just a hiccup I encountered, so thought it might be nice to fix it. But I see your point.